UPGRADING.md

```diff
@@ -173,7 +173,7 @@ conforming to standard library logging conventions.
 
 #### Misc.
 
-Un-exported `NewDeadlineTransport` and `ApiRequest`, which never should have been exported in the
+Un-exported `ApiRequest`, which never should have been exported in the
```

Comment:
you can leave this as it was; it's still true that `NewDeadlineTransport` was un-exported (and then now has been un-existed ;)